src/modules/utils.js

```diff
@@ -8,6 +8,17 @@ const emptyLogger = {
   log: empty
 };
 
+let uuidV4;
+const crypto = require('crypto');
```

Comment:
we can't use uuidd from `crypto` package yet, since this module still retains support for node 12

Reply:
this is why I check in line 14 if the function is defined...

Reply:
right, so if you keep the `uuid` package anyway, why doing this magic, lets stick with single version andd once we deprecate node 12 we will move to version from `crypto` package and remove the `uuid` module. I prefer to stay consistent between versions, the performance gain is not that crucual.

Reply:
@rluvaton just update to uuid version 9.. it will `crypto.randomUUID` if it's available (https://github.com/uuidjs/uuid/pull/600)